lib/readwise/client.rb

```diff
@@ -21,6 +23,18 @@ def create_highlights(highlights: [])
       raise NotImplementedError
     end
 
+    def get_highlight(highlight_id:)
+      url = BASE_URL + 'highlights/' + highlight_id
+
+      begin
+        res = make_readwise_request(url)
+      rescue => exception
+        raise exception
+      end
```

Comment:
@andjosh wondering if a begin rescue is the best way to handle errors that might bubble up when we raise them within the enclosed method.

Reply:
There's no need to begin/rescue if we're just going to raise again - this can just be: `res = make_readwise_request(url)`